Resources/Maps/Shuttles/Mushroom.yml

```diff
@@ -0,0 +1,5827 @@
+    - rot: -1.5707963267948966 rad
+      pos: -4.5,-9.5
+      parent: 1
+      type: Transform
+- proto: WallSolidDiagonal
+  entities:
+  - uid: 822
+    components:
+    - rot: 1.5707963267948966 rad
+      pos: -4.5,-21.5
+      parent: 1
+      type: Transform
+  - uid: 823
+    components:
+    - pos: -3.5,2.5
+      parent: 1
+      type: Transform
+  - uid: 824
+    components:
+    - rot: -1.5707963267948966 rad
+      pos: 1.5,2.5
+      parent: 1
+      type: Transform
+  - uid: 825
+    components:
+    - rot: 3.141592653589793 rad
+      pos: 2.5,-21.5
+      parent: 1
+      type: Transform
+  - uid: 826
+    components:
+    - rot: 3.141592653589793 rad
+      pos: 1.5,-22.5
+      parent: 1
+      type: Transform
+  - uid: 827
+    components:
+    - rot: 1.5707963267948966 rad
+      pos: -5.5,-20.5
+      parent: 1
+      type: Transform
+  - uid: 828
+    components:
+    - rot: 1.5707963267948966 rad
+      pos: -8.5,-18.5
+      parent: 1
+      type: Transform
+  - uid: 829
+    components:
+    - rot: 3.141592653589793 rad
+      pos: 7.5,-8.5
+      parent: 1
+      type: Transform
+  - uid: 830
+    components:
+    - rot: 3.141592653589793 rad
+      pos: 3.5,-20.5
+      parent: 1
+      type: Transform
+  - uid: 831
+    components:
+    - rot: 1.5707963267948966 rad
+      pos: -9.5,-8.5
+      parent: 1
+      type: Transform
+  - uid: 832
+    components:
+    - rot: 3.141592653589793 rad
+      pos: -3.5,-13.5
+      parent: 1
+      type: Transform
+  - uid: 833
+    components:
+    - rot: 1.5707963267948966 rad
+      pos: 1.5,-13.5
+      parent: 1
+      type: Transform
+  - uid: 834
+    components:
+    - pos: -9.5,-3.5
+      parent: 1
+      type: Transform
+  - uid: 835
+    components:
+    - rot: -1.5707963267948966 rad
+      pos: 7.5,-3.5
+      parent: 1
+      type: Transform
+  - uid: 836
+    components:
+    - rot: 3.141592653589793 rad
+      pos: 6.5,-18.5
+      parent: 1
+      type: Transform
+  - uid: 837
+    components:
+    - rot: 3.141592653589793 rad
+      pos: 4.5,-19.5
+      parent: 1
+      type: Transform
+  - uid: 838
+    components:
+    - rot: 1.5707963267948966 rad
+      pos: -6.5,-19.5
+      parent: 1
+      type: Transform
+  - uid: 839
+    components:
+    - rot: 1.5707963267948966 rad
+      pos: -3.5,-22.5
+      parent: 1
+      type: Transform
+  - uid: 840
+    components:
+    - rot: 3.141592653589793 rad
+      pos: 3.5,-9.5
+      parent: 1
+      type: Transform
+  - uid: 841
+    components:
+    - rot: 1.5707963267948966 rad
+      pos: -5.5,-9.5
+      parent: 1
+      type: Transform
+- proto: WallWeaponCapacitorRecharger
+  entities:
+  - uid: 842
+    components:
+    - rot: 3.141592653589793 rad
+      pos: 0.5,-2.5
+      parent: 1
+      type: Transform
+- proto: WarpPointShip
+  entities:
+  - uid: 868
+    components:
+    - rot: 1.5707963267948966 rad
+      pos: -0.5,-6.5
+      parent: 1
+      type: Transform
+- proto: WeaponRevolverMateba
```

Comment:
Detective gun?

Reply:
also on the sprinter

Reply:
Getting redone with rework.

Reply:
and the gourd as an Krammar (sec gun) and the anchor has MK38s (sec guns)

Reply:
Ill add it to the list, we will probably end up adding new safes to just allow ships to pick from to keep some baseline